src/value.zig

```diff
@@ -1999,6 +1999,11 @@ pub const Value = extern union {
                 }
                 return true;
             },
```

Comment:
This condition is reversed:
```zig
test "nan * 0 == nan" {
    const nan_times_zero = comptime std.math.nan(f64) * 0;
    try std.testing.expect(std.math.isNan(nan_times_zero));
}
```

Reply:
i think fixing it is out of scope for this PR...

Reply:
i think need a PR to modify all operation with NaN/Inf..